linera-rpc/tests/staged/formats.yaml

```diff
@@ -560,7 +560,7 @@ OutgoingMessage:
     - authenticated_signer:
         OPTION:
           TYPENAME: Owner
-    - is_skippable: BOOL
+    - attributes: BOOL
```

Comment:
Why did `attributes` turn into a single `BOOL` here? Also in the TypeScript code?